apps/ledger-live-mobile/src/components/CustomImage/useCenteredImage/getCenteredCropParams.ts

```diff
@@ -0,0 +1,27 @@
+import { ImageCropError } from "@ledgerhq/live-common/customImage/errors";
+import { ImageDimensions } from "../types";
+
+/**
+ * Returns the parameters to crop an image in the center.
+ * @param imageDimensions The dimensions of the image to crop
+ * @param cropDimensions The dimensions of the container to crop in
+ * @returns The parameters to crop the image in the center
+ */
+
+export function getCenteredCropParams(
+  imageDimensions: ImageDimensions,
+  cropDimensions: ImageDimensions,
+): { originX: number; originY: number; width: number; height: number } {
```

Comment:
[COULD] create a type for `{ originX: number; originY: number; width: number; height: number }`